.buildkite/pipeline.yml

```diff
@@ -135,6 +135,25 @@ steps:
       - replayio/buildevents#adb8a05: ~
     command: "be_cmd metabase-tests -- /home/ubuntu/chromium/src/replay_build_scripts/metabase.sh"
 
+  - label: "Trigger Devtools E2E Tests"
+    key: "devtools-test-suite"
+    soft_fail: true
+    depends_on: "build-chromium-linux-x86_64"
+    agents:
+      - "runtimeType=chromiumbuild"
+      - "os=linux"
```

Comment:
This could probably run on almost any buildkite agent, but for now this is fine

Reply:
ah yeah, I ran into a problem on the first commit up above, and added this as a test.  let me see if I can figure out what was going on in there.

Reply:
ah yeah, the script wasn't found in the first commit.  might this have been because of `skip-checkout` usage, and that step was run on a build instance that didn't have the script already?

Reply:
oh! Right, good point. If we move this script in to backend then it could run on any buildkite agent but not a big deal.